prometheus-jmx-exporter.yaml

```diff
@@ -0,0 +1,52 @@
+package:
+  name: prometheus-jmx-exporter
+  version: 1.0.1
+  epoch: 0
+  url: https://github.com/prometheus/jmx_exporter
+  description: JMX to Prometheus exporter javaagent
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - bash
+      - busybox
+      - ca-certificates-bundle
+      - maven
+      - openjdk-11
+      - openjdk-11-default-jvm
+  environment:
+    LANG: en_US.UTF-8
+    JAVA_HOME: /usr/lib/jvm/default-jvm
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/prometheus/jmx_exporter
+      tag: ${{package.version}}
+      expected-commit: 328d8a36cd4b81c99d19c7490a78f49ddf6fda24
+
+  - name: build
+    runs: |
+      mvn -pl jmx_prometheus_javaagent,jmx_prometheus_httpserver -am package
+
+  - name: install
+    runs: |
+      mkdir -p ${{targets.destdir}}/usr/lib/prometheus
+      mv jmx_prometheus_javaagent/target/jmx_prometheus_javaagent-${{package.version}}.jar ${{targets.destdir}}/usr/lib/prometheus/jmx_prometheus_javaagent.jar
+
+subpackages:
+  - name: ${{package.name}}-httpserver
+    description: "JMX to Prometheus exporter httpserver"
+    pipeline:
+      - name: install
+        runs: |
+          mkdir -p ${{targets.subpkgdir}}/usr/lib/prometheus
+          mv jmx_prometheus_httpserver/target/jmx_prometheus_httpserver-${{package.version}}.jar ${{targets.subpkgdir}}/usr/lib/prometheus/jmx_prometheus_httpserver.jar
+
+update:
+  enabled: true
+  github:
+    identifier: prometheus/jmx_exporter
+    strip-prefix: parent-
```

Comment:
```suggestion
    strip-prefix: parent-

test:
  environment:
    contents:
      packages:
        - openjdk-11
        - openjdk-11-default-jvm
        - ${{package.name}}-httpserver
  pipeline:
    - name: Create a sample config files
      runs: |
        echo -e "rules:\n- pattern: \".*\"" > agent-config.yaml
        echo -e "hostPort: localhost:1234\nrules:\n- pattern: \".*\"" > server-config.yaml
    - name: Verify agent jar
      runs: |
        java -javaagent:/usr/lib/prometheus/jmx_prometheus_javaagent.jar=1234:agent-config.yaml --version
    - name: Verify httpserver jar
      runs: |
        java -jar /usr/lib/prometheus/jmx_prometheus_httpserver.jar 1234 server-config.yaml > output.log 2>&1 &
        sleep 10
        if ! grep -q 'io.prometheus.jmx.WebServer | Running' output.log; then
          echo "Expected log message not found" >&2
          cat output.log
          exit 1
        fi

```

Reply:
Thanks for the contribution !
Works great on my local computer, i commit the change